### PR TITLE
[OSDOCS-5271]: Fix AWS STS terms

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.9+

Issue:
[OSDOCS-5271](https://issues.redhat.com//browse/OSDOCS-5271)

Link to docs preview:
See https://github.com/openshift/openshift-docs/pull/55728#pullrequestreview-1290019036

QE review:
- [ ] QE has approved this change.
IMO no QE required.

Additional information:

### CODE REVIEW

**Comment by bmcelvee** [comment]
LGTM!

File: authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc [hunk 1]
```diff
@@ -26,7 +26,7 @@ Mint mode is the default and recommended best practice setting for the CCO to us
 
 * **xref:../../authentication/managing_cloud_provider_credentials/cco-mode-manual.adoc#cco-mode-manual[Manual]**: In manual mode, a user manages cloud credentials instead of the CCO.
 
-** **xref:../../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#cco-mode-sts[Manual with AWS Secure Token Service]**: In manual mode, you can configure an AWS cluster to use Amazon Web Services Secure Token Service (AWS STS). With this configuration, the CCO uses temporary credentials for different components.
```
Comment (jeana-redhat) [inline authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc:29]: https://55728--docspreview.netlify.app/openshift-enterprise/latest/authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.html#about-cloud-credential-operator-modes

File: authentication/managing_cloud_provider_credentials/cco-mode-manual.adoc [hunk 2]
```diff
@@ -17,7 +17,7 @@ For information about configuring your cloud provider to use manual mode, see _M
 [id="manual-mode-sts-blurb"]
 == Manual mode with AWS STS
 
-You can configure an AWS cluster in manual mode to use xref:../../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#cco-mode-sts[Amazon Web Services Secure Token Service (AWS STS)]. With this configuration, the CCO uses temporary credentials for different components.
```
Comment (jeana-redhat) [inline authentication/managing_cloud_provider_credentials/cco-mode-manual.adoc:20]: https://55728--docspreview.netlify.app/openshift-enterprise/latest/authentication/managing_cloud_provider_credentials/cco-mode-manual.html#manual-mode-sts-blurb

File: authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc [hunk 3]
```diff
@@ -1,6 +1,6 @@
 :_content-type: ASSEMBLY
 [id="cco-mode-sts"]
-= Using manual mode with Amazon Web Services Secure Token Service
```
Comment (jeana-redhat) [inline authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc:3]: https://55728--docspreview.netlify.app/openshift-enterprise/latest/authentication/managing_cloud_provider_credentials/cco-mode-sts.html

File: modules/bootstrap-aws-load-balancer-operator.adoc [hunk 4]
```diff
@@ -3,7 +3,7 @@
 
 :_content-type: PROCEDURE
 [id="nw-bootstra-albo-on-sts-cluster_{context}"]
-= Bootstrapping AWS Load Balancer Operator on Secure Token Service cluster
```
Comment (jeana-redhat) [inline modules/bootstrap-aws-load-balancer-operator.adoc:6]: https://55728--docspreview.netlify.app/openshift-enterprise/latest/networking/aws_load_balancer_operator/installing-albo-sts-cluster.html

File: modules/configuring-albo-on-sts-cluster-predefined-credentials.adoc [hunk 5]
```diff
@@ -3,7 +3,7 @@
 
 :_content-type: PROCEDURE
 [id="nw-installing-albo-on-sts-cluster-predefined-credentials_{context}"]
-= Configuring the AWS Load Balancer Operator on Secure Token Service cluster by using specific credentials
```
Comment (jeana-redhat) [inline modules/configuring-albo-on-sts-cluster-predefined-credentials.adoc:6]: https://55728--docspreview.netlify.app/openshift-enterprise/latest/networking/aws_load_balancer_operator/installing-albo-sts-cluster.html

File: modules/configuring-albo-on-sts-cluster.adoc [hunk 6]
```diff
@@ -3,7 +3,7 @@
 
 :_content-type: PROCEDURE
 [id="nw-installing-albo-on-sts-cluster_{context}"]
-= Configuring AWS Load Balancer Operator on Secure Token Service cluster by using managed `CredentialsRequest` objects
```
Comment (jeana-redhat) [inline modules/configuring-albo-on-sts-cluster.adoc:6]: https://55728--docspreview.netlify.app/openshift-enterprise/latest/networking/aws_load_balancer_operator/installing-albo-sts-cluster.html

File: modules/persistent-storage-csi-efs-sts.adoc [hunk 7]
```diff
@@ -7,9 +7,9 @@
 
 :_content-type: PROCEDURE
 [id="efs-sts_{context}"]
-= Configuring AWS EFS CSI Driver Operator with Secure Token Service
+= Configuring AWS EFS CSI Driver Operator with Security Token Service
```
Comment (jeana-redhat) [inline modules/persistent-storage-csi-efs-sts.adoc:10]: https://55728--docspreview.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-aws-efs.html#efs-sts_persistent-storage-csi-aws-efs

File: rosa_planning/rosa-sts-ocm-role.adoc [hunk 8]
```diff
@@ -17,19 +17,19 @@ The AWS IAM roles required to use {cluster-manager} are:
...
-* `Installer-Role` 
+* `Worker-Role`
+* `Support-Role`
+* `Installer-Role`
 * `ControlPlane-Role`
 
 [NOTE]
 ====
-Role creation does not request your AWS access or secret keys. AWS Secure Token Service (STS) is used as the basis of this workflow. AWS STS uses temporary, limited-privilege credentials to provide authentication.
+Role creation does not request your AWS access or secret keys. AWS Security Token Service (STS) is used as the basis of this workflow. AWS STS uses temporary, limited-privilege credentials to provide authentication.
```
Comment (jeana-redhat) [inline rosa_planning/rosa-sts-ocm-role.adoc:27]: IDK how to get ROSA docs previews :/

**Review by jeana-redhat — COMMENTED** [review]
This is a find and replace so it shouldn't break anything but here are some previews

File: networking/aws_load_balancer_operator/installing-albo-sts-cluster.adoc [hunk 9]
```diff
@@ -1,12 +1,12 @@
 :_content-type: ASSEMBLY
 [id="albo-sts-cluster"]
-= Installing AWS Load Balancer Operator on Secure Token Service cluster
+= Installing AWS Load Balancer Operator on Security Token Service cluster
```
Comment (bscott-rh) [inline networking/aws_load_balancer_operator/installing-albo-sts-cluster.adoc:3]: Should this be "a Security Token Service cluster"?

**Review by bscott-rh — APPROVED** [review]
one nit/question, otherwise LGTM.

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: networking/aws_load_balancer_operator/installing-albo-sts-cluster.adoc (hunk 9 above)
Comment (jeana-redhat) [reply networking/aws_load_balancer_operator/installing-albo-sts-cluster.adoc:3]: Probably :)

**Pull request merged**